app/controllers/comments_controller.rb

```diff
@@ -1,2 +1,18 @@
 class CommentsController < ApplicationController
```

Comment:
- [ ] The comment controller should contain a new action, kindly consider implementing this so your app has a new action for comments
e.g

```ruby
def new
    @comment = Comment.new
  end
```